test/yargs.js

```diff
@@ -324,6 +348,22 @@ describe('yargs dsl tests', function () {
       r.logs.join(' ').should.match(/Choose yer command:/)
     })
 
+    it('handles a missing locale', function () {
+      loadLocale('zz_ZZ.UTF-8')
+
+      var r = checkOutput(function () {
+        yargs(['snuh', '-h'])
+          .command('blerg', 'blerg command')
+          .help('h')
+          .wrap(null)
+          .argv
+      })
+
+      yargs.locale().should.equal('zz')
```

Comment:
This is an interesting one, do we perhaps just need to add one more rule, here's the listing of:

`locale -a`

```
no_NO.ISO8859-1
no_NO.ISO8859-15
no_NO.UTF-8
pl_PL
pl_PL.ISO8859-2
pl_PL.UTF-8
pt_BR
pt_BR.ISO8859-1
pt_BR.UTF-8
pt_PT
pt_PT.ISO8859-1
pt_PT.ISO8859-15
pt_PT.UTF-8
```

Should y18n ignore everything after a `.`, `zz_ZZ.UTF-8` becomes `zz_ZZ`.